### ISSUE TITLE
Problems in add scope to Auth0

### ISSUE THREAD

**Issue by guidiego** [auth0/nextjs-auth0#322]
### Description
Hi! Today I am migrating myself to nextjs-auth0 and found trouble :) My actual code is based on '@auth0/auth0-spa-js' where I use Auth0Client

```typescript
const auth0 = new Auth0Client({
  client_id: config.AUTH0_CLIENT_ID,
  domain: config.AUTH0_DOMAIN,
  redirect_uri: config.AUTH0_REDIRECT_URI,
  audience: config.AUTH0_AUDIENCE,
  scope:
    'all my scopes here',
  useRefreshTokens: true,
});
```

I tried to convert it into this code:

```typescript
    const { accessToken: token } = await getAccessToken(ctx.req, ctx.res, {
      refresh: true,
      scopes: [ 'all', 'my', 'scopes'],
    });
```

For some reason this throws me an error: 

```
AccessTokenError: Could not retrieve an access token with scopes "openid profile email offline_access read:dashboard-data read:dashboard-notifications write:dashboard-comment view:dashboard-view edit:dashboard-users access:dashboard-api". The user will need to sign in again.
    at Object.<anonymous> (/Users/guidiego/workspace/streem.ai/app/node_modules/@auth0/nextjs-auth0/dist/session/get-access-token.js:29:23)
    at Generator.next (<anonymous>)
    at /Users/guidiego/workspace/streem.ai/app/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/guidiego/workspace/streem.ai/app/node_modules/tslib/tslib.js:113:16)
    at Object.getAccessToken (/Users/guidiego/workspace/streem.ai/app/node_modules/@auth0/nextjs-auth0/dist/session/get-access-token.js:11:54)
    at getAccessToken (/Users/guidiego/workspace/streem.ai/app/node_modules/@auth0/nextjs-auth0/dist/index.js:56:51)
    at eval (webpack-internal:///./src/utils/server/nextBaseSSP.ts:14:89)
    at renderToHTML (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/render.js:40:221)
    at async /Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:107:97
    at async /Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:100:142
    at async DevServer.renderToHTMLWithComponents (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:132:387)
    at async DevServer.renderToHTML (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:133:522)
    at async DevServer.renderToHTML (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/server/next-dev-server.js:34:578)
    at async DevServer.render (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:72:236)
    at async Object.fn (/Users/guidiego/workspace/streem.ai/app/node_modules/next/dist/next-server/server/next-server.js:56:580) {
  code: 'insufficient_scope'
}
```

For my understanding, the `scope` from `getAccessToken` should be a part of the whole scope that I provide before. I have tried added on `/api/auth/[...auth0].js` too:

```typescript
export default handleAuth({
  async login(req, res) {
    await handleLogin(req, res, {
      authorizationParams: {
        scopes: [ 'all', 'my', 'scopes' ],
      },
    });
  },
});
```

But nothing changes 

### Reproduction
- Quick start nextjs
- Try to add scope ass the previous explanation 

### Environment
- **Version of this library used:** Latest
- **Other relevant versions (language, server software, OS, browser):** macOS
- **Other modules/plugins/libraries that might be involved:** nextjs

**Comment by adamjmcgrath** [comment]
Hi @guidiego - you were right to request scopes at login time, you just got the wrong name for the property, like spa js - it's `scope` not `scopes`

You can define it in the login options as a string ([more info](https://auth0.github.io/nextjs-auth0/interfaces/config.authorizationparameters.html#scope))

```js
// pages/api/auth/[...auth0].js
export default handleAuth({
  async login(req, res) {
    await handleLogin(req, res, {
      authorizationParams: {
        scope: 'all my scopes',
      },
    });
  },
});
```

Or, if you don't want to override the builtin login handler - you can specify it in the environment variable ([more info](https://auth0.github.io/nextjs-auth0/modules/config.html)):

```sh
# .env.local
AUTH0_SCOPE="all my scopes"
```

**Comment by mustafaKamal-fe** [comment]
Regarding this matter, i have a couple of questions:
First lets say that: 

1. In my Auth0 dashboard i created a custom API with some speacial permissions and roles
2. I enabled RBAC (role based access control) 
3. I enabled `include permissions in access tokens` options as well

the questions are: 

1. Do i still need to include the scopes inside 
  `export default initAuth0({..., autherizationParams: {audience: process.env.AUDIENCE, scope: 'my_special_scopes_here'}})`
2.Why when in the code above, when i include `openid` as the only scope, i still see the logged in user speacial scopes i gave? is it because of the options i have enabled in my application inside auth0 dashboard configuration.
3.Final question, what is the best practise in this SDK to check if a user holds a special permission. something similar to `express-jwt` function `checkJwt`?

Again Thank you so much for this massive work.
